docs/Contributing-back.rst

```diff
@@ -0,0 +1,610 @@
+in a large code base that is constantly being updated by a worldwide community.
+
+This also applies to easyconfig files, where we try to maintain a strict style
+that mostly matches the established PEP8 coding style for Python (since
+easyconfigs are written in Python syntax). However, also the grouping and
+ordering of easyconfig parameters is a part of the 'code' style we maintain.
+
+
+.. _contributing_back_review_process_review_pr:
+
+Comparing with existing easyconfigs (``--review-pr``)
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+We try to maintain **consistency across easyconfig files** for a particular
+software package, across multiple software versions, toolchains and variants
+(with a different ``versionsuffix``).
+
+Therefore, easyconfig contributions are also reviewed using ``eb --review-pr <PR #>``,
+which compares the touched easyconfig files to those in the current
+``develop`` branch that are most closely related.
+
+The easyconfig files to compare with are selected based on similarity,
+by combining two criteria, in order.
+
+First, the software version is taken into account, using one of the following
+criteria:
+
+* exact match on software version match
+* match on major/minor software version
+* match on major software version
+* no match on software version
+
+This is combined with one of the criteria below (in order):
+
+* matching versionsuffix and toolchain name/version
+* matching versionsuffix and toolchain name (any toolchain version)
+* matching versionsuffix (any toolchain name/version)
+* matching toolchain name/version (any versionsuffix)
+* matching toolchain name (any versionsuffix, toolchain version)
+* no extra requirements (any versionsuffix, toolchain name/version)
+
+The first combination of one of the software version criteria with one of
+the other criteria that yields one or more matching easyconfig files is used.
+If none of the combinations match, no easyconfig files for that particular software
+package are available yet, and no comparison is made.
+
+The output of ``--review-pr`` provides a 'multidiff' comparison, which highlights
+the differences between the easyconfig file in the pull request and the most similar
+selected ones from the current ``develop`` branch.
+
+For example:
+
+.. raw:: html
+
+  <script type="text/javascript" src="https://asciinema.org/a/103889.js" id="asciicast-103889" async></script>
+
+Interpreting this output is a quick and easy way to assess how different the
+contributed easyconfig files are from the existing easyconfigs, although it
+does require a bit of practice because of the density of the provided information.
+
+
+.. _contributing_back_review_process_test_reports:
+
+Test reports for easyconfig contributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For easyconfig contributions, an accompanying **test report must be submitted** to confirm that the touched easyconfig files (still) work as expected.
```

Comment:
preferably by the contributor, but this can also be done by a 3rd party

I'll clarify that, thanks